docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -124,6 +124,10 @@ values. Typical usage does not require you to configure them.
 * - `delta.checkpoint-row-statistics-writing.enabled`
   - Enable writing row statistics to checkpoint files.
   - `true`
+* - ``delta.checkpoint-filtering.enabled``
```

Comment:
Could you pls test coverage into `TestDeltaLakeFileOperations` with `checkpoint_filtering_enabled` session property enabled to add more transparence in regards to the consequences coming with this change?